### PR TITLE
Add nflow_executor.stopped field

### PR DESCRIPTION
**Pull request by jsyrjala**
Add stopped field to nflow_executor table. Added corresponding field to REST API and UI.

This change requires at least a minor version number bump (-> 5.2.0).

### CODE REVIEW

File: nflow-engine/src/main/resources/scripts/db/postgresql.create.ddl.sql [hunk 1]
```diff
@@ -75,7 +75,8 @@ create table if not exists nflow_executor (
   executor_group varchar(64),
   started timestamptz not null default current_timestamp,
   active timestamptz not null,
-  expires timestamptz not null
+  expires timestamptz not null,
+  stopped timestamp
```
Comment (efonsell) [inline nflow-engine/src/main/resources/scripts/db/postgresql.create.ddl.sql]:
```suggestion
  stopped timestamptz
```

File: nflow-engine/src/main/resources/scripts/db/update-5.2.0-x/postgresql.update.ddl.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+alter table nflow_explorer add stopped timestamp;
```
Comment (efonsell) [inline nflow-engine/src/main/resources/scripts/db/update-5.2.0-x/postgresql.update.ddl.sql]:
```suggestion
alter table nflow_explorer add stopped timestamptz;
```

Comment (jsyrjala) [reply nflow-engine/src/main/resources/scripts/db/update-5.2.0-x/postgresql.update.ddl.sql]: Fixed

File: nflow-engine/src/main/resources/scripts/db/postgresql.create.ddl.sql (hunk 1 above)
Comment (jsyrjala) [reply nflow-engine/src/main/resources/scripts/db/postgresql.create.ddl.sql]: Fixed

File: nflow-rest-api-common/src/test/java/io/nflow/rest/v1/converter/ListWorkflowExecutorConverterTest.java [hunk 3]
```diff
@@ -20,7 +20,7 @@
   @Test
   public void convertWorks() {
     WorkflowExecutor executor = new WorkflowExecutor(1, "host", 2, "executorGroup", now(), now().plusMinutes(1), now()
-        .plusMinutes(15));
+        .plusMinutes(15), now().plusSeconds(-7));
```
Comment (efonsell) [inline nflow-rest-api-common/src/test/java/io/nflow/rest/v1/converter/ListWorkflowExecutorConverterTest.java]:
```suggestion
        .plusMinutes(15), now().minusSeconds(7));
```

Comment (jsyrjala) [reply nflow-rest-api-common/src/test/java/io/nflow/rest/v1/converter/ListWorkflowExecutorConverterTest.java]: fixed

**Pull request merged**